src/main/java/com/arpnetworking/metrics/mad/parsers/ProtobufV1ToRecordParser.java

```diff
@@ -42,7 +42,8 @@
  *
  * @author Brandon Arp (brandon dot arp at inscopemetrics dot com)
  */
-public class ProtobufToRecordParser implements Parser<List<Record>, HttpRequest> {
+@SuppressWarnings("deprecation")
```

Comment:
Please add the javadoc annotation for deprecation as well.

Reply:
Also should this be a `final` class?

Reply:
I didn't deprecate the class, I'm consuming a class that has been deprecated.  In this case, the v1 messages.

Reply:
Made final

Reply:
Sorry I clearly wasnted thinking on that!